src/ImageSearchModal.jsx

```diff
@@ -125,7 +125,7 @@ export class ImageSearchModal extends React.Component {
             <>
                 { this.props.userServiceAvailable && this.props.systemServiceAvailable &&
                     <form className="ct-form">
-                        <label className="control-label" htmlFor="as-user">{_("Download as:")}</label>
+                        <label className="control-label" htmlFor="as-user">{_("Download to")}</label>
```

Comment:
Why 'Download to'?
Download to I would expect to be a label for a location where the images are stored.  This is for the owner of the image.

Reply:
@garrett

Reply:
It downloads to the system store or the user's store?

Reply:
Although, I guess it's really more about running as a system or rootless user container, than downloading at all? (But this dialog is about searching and downloading an image, of course.)

Reply:
There is the concept of stores as in separate storage locations for different users. But still 'Download to: garrett' does not make sense to me, it could be 'Download to: /storage' but 'Download to: garett' seems wrong seems there is no garrett storage/store call it how you want. But a default image location configured for garrett user.

Reply:
Downloading _as_ system or a user doesn't necessarily imply where it's downloaded or that you can only run it as system or user after downloading. That's the problem I have with "Downloading as". Additionally, Cockpit is doing the downloading (one way or the other).

I guess "Downloading as" or "Downloading to" are both a bit problematic here. I figured "to" at least indicates where it's going.